Formula/v/valkey.rb

```diff
@@ -0,0 +1,49 @@
+class Valkey < Formula
+  desc "High-performance data structure server that primarily serves key/value workloads"
+  homepage "https://valkey.io"
+  url "https://github.com/valkey-io/valkey/archive/refs/tags/redis-7.2.4.tar.gz"
```

Comment:
The correct tag is 7.2.4-rc1. The current tag that is listed will just install Redis, since it points to the start of the fork. I tested it locally and it does indeed just seem to install Redis. I just briefly looked into this, but happy to help more if needed.

Reply:
Thanks for stopping by and giving us a heads up @madolson. I will get this fixed shortly.

Reply:
Cool. Let me know if there is there is a better process for keeping these formulas up to date.

Reply:
`valkey` has been added to our autobump list, which will check for a new release every few hours and open a PR. If you'd like to open PR's yourself as part of your release workflow instead, I can help with that. Just let me know.